jina/peapods/pea.py

```diff
@@ -365,8 +365,11 @@ def post_init(self):
     def close(self) -> None:
         """Gracefully close this pea and release all resources """
         if self.is_ready.is_set() and hasattr(self, 'ctrl_addr'):
-            return send_ctrl_message(self.ctrl_addr, jina_pb2.Request.ControlRequest.TERMINATE,
-                                     timeout=self.args.timeout_ctrl)
+            send_ctrl_message(self.ctrl_addr, jina_pb2.Request.ControlRequest.TERMINATE,
```

Comment:
does it make sense sending it before or after tearing down? maybe the order should be reversed

Reply:
Yes, I think it makes sense to send TERMINATE request after teardown is complete

Reply:
better not change it until you fully understand the current eventloop with the knowledge I pointed here https://github.com/jina-ai/jina/pull/905#issuecomment-686640530